hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/execution/HederaMessageCallProcessorV038.java

```diff
@@ -153,6 +154,10 @@ private void haltFrameAndTraceCreationResult(
             final MessageFrame frame, final OperationTracer operationTracer, final ExceptionalHaltReason haltReason) {
         frame.decrementRemainingGas(frame.getRemainingGas());
         frame.setState(EXCEPTIONAL_HALT);
-        operationTracer.traceAccountCreationResult(frame, Optional.of(haltReason));
+        final var wrappedHaltReason = Optional.of(haltReason);
```

Comment:
Too bad we have to see this code twice.  Along with other code here in `...V038`.  I guess since there's no mixins in java it's unavoidable.  (I dislike using default methods in interfaces to accomplish this.)

Reply:
True, will only be one change in modularized `ContractService`.